wp-includes/sqlite-ast/class-wp-sqlite-driver.php

```diff
@@ -2182,7 +2236,7 @@ private function translate( $node ): ?string {
 					'%s AS %s',
 					$value,
 					$this->quote_sqlite_identifier(
-						'@@' . ( $type_token ? "$type_token->value." : '' ) . $original_name
+						'@@' . ( $type_token ? "{$type_token->get_value()}." : '' ) . $original_name
```

Comment:
nit: slightly clearer code style

```suggestion
						'@@' . ( $type_token ? $type_token->get_value().'' : '' ) . $original_name
```